viewing_party/party.py

```diff
@@ -1,23 +1,212 @@
+
 # ------------- WAVE 1 --------------------
 
+
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    if not title or not genre or not rating:
+        return None
+    new_movie["title"] = title
+    new_movie["genre"] = genre
+    new_movie["rating"] = rating
+    
+    return new_movie
+
+
+def add_to_watched(user_data, movie):
+    user_data["watched"] = []
+    user_data["watched"].append(movie)
+
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"] = []
+    user_data["watchlist"].append(movie)
+
+    return user_data
+
+
+def watch_movie(user_data, title):
+    for value in range(len(user_data["watchlist"])):
+        
+        if user_data["watchlist"][value]["title"] == title:
+            move_to_watched = user_data["watchlist"][value]

```

Comment:
You could shorten this by simply iterating over the `user_data["watchlist"]` and checking for the `movie["title"]`. Something like this:
```
def watch_movie(user_data, title):
    for movie in user_data["watchlist"]:
        if movie["title"] == title:
            user_data["watchlist"].remove(movie)
            user_data["watched"].append(movie)

    return user_data